tests/network/network_simplification/test_network_graph_simplificator.py

```diff
@@ -132,26 +139,30 @@ def _get_network_simplification_with_attribute_exclusion(
     def _get_nx_digraph_factory(self) -> Iterator[Callable[[], nx.MultiDiGraph]]:
         def create_nx_multidigraph():
             _nx_digraph = nx.MultiDiGraph()
-            for i in range(1, 16):
+            for i in range(1, 19):
                 _nx_digraph.add_node(i, x=i, y=i * 10)
 
-            _nx_digraph.add_edge(1, 2, a=np.nan)
-            _nx_digraph.add_edge(2, 1, a=np.nan)
-            _nx_digraph.add_edge(2, 3, a=np.nan)
-            _nx_digraph.add_edge(3, 4, a=np.nan)
+            _nx_digraph.add_edge(1, 2, a='None')
```

Comment:
What is the use of attribute a?

Reply:
to activate the simplification with attribute. a can be bridge, tunnel, x, y, ... any attribute of an edge.

Reply:
Can it be renamed to something more declarative?

Reply:
Done